src/common/intl/messages.js

```diff
@@ -1108,14 +1102,12 @@ var messages = {
   serverError: 'Error',
   serverAddFailed: 'Adding server failed',
   serverStatus: 'Status',
-  serverConnectionFailed: 'Connection failed. Click for more information.',
+  serverConnectionFailed: 'Connection failed',
   serverConnecting: 'Connecting...',
   serverConnected: 'Connected',
   serverDisconnected: 'Disconnected',
   serverAuthFailed: 'Authentication error',
   serverUnknownError: 'Unknown error',
-  serverSelfSignedCertError: 'Invalid self-signed certificate',
-  serverSelfSignedCertQuestion: 'Do you want to accept self-signed certificate for this server even though it would decrease security?',
```

Comment:
Your rebase did not went well, as you can see, you have removed a lot of unrelated changes.